templates/saphana.tpl.yaml

```diff
@@ -0,0 +1,188 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: {{ os }}-{{ item.workload }}-{{ item.flavor }}
+  annotations:
+    openshift.io/display-name: "Red Hat Enterprise Linux 8.4 Beta VM for SAP HANA workloads"
+    description: >-
+      Template for Red Hat Enterprise Linux 8.4 for SAP HANA workloads.
+      Please consult the SAP HANA guide for node setup requirements.
+    tags: "hidden,kubevirt,virtualmachine,linux,rhel,sap,hana"
+    iconClass: "icon-{{ icon }}"
+    openshift.io/provider-display-name: "KubeVirt"
+    openshift.io/documentation-url: "https://github.com/kubevirt/common-templates"
+    openshift.io/support-url: "https://github.com/kubevirt/common-templates/issues"
+    template.openshift.io/bindable: "false"
+    template.kubevirt.io/version: v1alpha1
+    defaults.template.kubevirt.io/disk: rootdisk
+{% for osl in oslabels %}
+    name.os.template.kubevirt.io/{{ osl }}: {{ lookup('osinfo', oslabels[ 0 ]).name }} or higher
+{% endfor %}
+  labels:
+{% for osl in oslabels %}
+    os.template.kubevirt.io/{{ osl }}: "true"
+{% endfor %}
+    workload.template.kubevirt.io/{{ item.workload }}: "true"
+    flavor.template.kubevirt.io/{{ item.flavor }}: "true"
+    template.kubevirt.io/type: "base"
+    template.kubevirt.io/version: "{{ lookup('env', 'VERSION') | default('devel', true) }}"
+{% if item.default %}
+    template.kubevirt.io/default-os-variant: "true"
+{% endif %}
+objects:
+  - apiVersion: kubevirt.io/v1
+    kind: VirtualMachine
+    metadata:
+      labels:
+        vm.kubevirt.io/template: {{ os }}-{{ item.workload }}-{{ item.flavor }}
+        vm.kubevirt.io/template.version: "{{ lookup('env', 'VERSION') | default('devel', true) }}"
+        vm.kubevirt.io/template.revision: "{{ lookup('env', 'REVISION') | default(1, true) }}"
+        app: ${NAME}
+      name: ${NAME}
+    spec:
+      dataVolumeTemplates:
+        - apiVersion: cdi.kubevirt.io/v1beta1
+          kind: DataVolume
+          metadata:
+            name: ${NAME}
+          spec:
+            storage:
+              resources:
+                requests:
+                  storage: 50Gi
+            source:
+              pvc:
+                name: ${SRC_PVC_NAME}
+                namespace: ${SRC_PVC_NAMESPACE}
+      running: false
+      template:
+        metadata:
+          annotations:
+            vm.kubevirt.io/os: "{{ os }}"
+            vm.kubevirt.io/workload: "{{ item.workload }}"
+            vm.kubevirt.io/flavor: "{{ item.flavor }}"
+          labels:
+            kubevirt.io/domain: ${NAME}
+            kubevirt.io/size: {{ item.flavor }}
+        spec:
+          hostname: ${NAME}
+          domain:
+            ioThreadsPolicy: auto
+            cpu:
+              cores: {{ '${{CPU_CORES}}' }}
```

Comment:
shouldn't this have syntax `${CPU_CORES}`?
Same for `threads` and `sockets`

Reply:
No, it should be filled in as numeric value and not as string: https://docs.openshift.com/container-platform/4.7/openshift_images/using-templates.html#templates-writing-parameters_using-templates